usage/app/lib/UsageRecorder.scala

```diff
@@ -77,20 +86,24 @@ class UsageRecorder(
         val result = func(mediaUsage)
         logger.info(
           logMarker,
-          s"'$opName' DB Operation for ${mediaUsage.grouping} -  on mediaID: ${mediaUsage.mediaId} with result: ${result}"
+          s"'$opName' DB Operation for ${mediaUsage.grouping} - on mediaID: ${mediaUsage.mediaId} with result: $result"
         )
         usageMetrics.incrementUpdated
         result
       }
 
-      val markAsRemovedOps = (dbUsages diff usageGroup.usages)
+      val toMarkAsRemoved = (dbUsageKeys diff streamUsageKeys).flatMap(dbUsageMap.get)
```

Comment:
I think this would be marginally more readable (and consistent with line 105) if it didn't use infix notation

Reply:
Same for Line 99 and 103

Reply:
done in https://github.com/guardian/grid/pull/3735/commits/4f88a93f12f955708569ea579eab4fade1a57d6e